numpy/__init__.pyi

```diff
@@ -479,24 +479,24 @@ class complex128(complexfloating):
 class flexible(_real_generic): ...  # type: ignore
 
 class void(flexible):
-    def __init__(self, value: Union[int, integer, bool_, bytes, bytes_]): ...
+    def __init__(self, __value: Union[int, integer, bool_, bytes, bytes_]): ...
 
 class character(_real_generic): ...  # type: ignore
 
 class bytes_(character):
     @overload
-    def __init__(self, value: object = ...) -> None: ...
+    def __init__(self, __value: object = ...) -> None: ...
     @overload
     def __init__(
-        self, value: object, encoding: str = ..., errors: str = ...
+        self, __value: object, encoding: str = ..., errors: str = ...
```

Comment:
The `encoding` and `errors` parameters in `np.bytes_` and `np.str_` are pretty much the only case where one *can* successfully supply a keyword argument.

Example
---------
``` python
>>> import numpy as np

>>> np.bytes_('bob', 'utf-8')
b'bob'

>>> np.bytes_('bob', 'utf-16')
b'\xff\xfeb\x00o\x00b'

>>> np.bytes_('bob', encoding='utf-16')
b'\xff\xfeb\x00o\x00b'
```